python.d/httpcheck.chart.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+# Description: http check netdata python.d module
+# Original Author: ccremer (github.com/ccremer)
+
+import urllib3
+import time
```

Comment:
python3 has `monotonic` which is [preferable](https://github.com/firehol/netdata/blob/master/plugins.d/python.d.plugin#L18-L21) for time management